app/Scripts/enemy.gd

```diff
@@ -40,14 +51,15 @@ func _process_walking(delta: float):
 func _on_timer_timeout() -> void:
 	match _current_state:
 		State.IDLE:
-			# choose direction and switch to walking
+			#choose direction and switch to walking
 			_walk_direction = _random_normalized_direction()
 			_current_state = State.WALKING
 			_timer.start(rand_range(2, 5))
+			pass
 		State.WALKING:
 			# stop walking
 			_current_state = State.IDLE
-			_timer.start(rand_range(2, 5))
+			#_timer.start(rand_range(2, 5))
```

Comment:
if we aren't spawning enemies then i think we should revert these changes. Fine to keep all the new images, but all the other changes that are unverifiable should ideally not be included in the PR